includes/slug-in-classname/slug-in-classname.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+//Page Slug Body Class
```

Comment:
I'm not sure this is a super valuable comment for a function called `add_slug_body_class`.